ts/src/base/Exchange.ts

```diff
@@ -1779,15 +1779,26 @@ export default class Exchange {
     parseToNumeric (number) {
         const stringVersion = this.numberToString (number); // this will convert 1.0 and 1 to "1" and 1.1 to "1.1"
         // keep this in mind:
-        // in JS: 1 == 1.0 is true
+        // in JS: 1 == 1.0 is true;  1 === 1.0 is true
         // in Python: 1 == 1.0 is true
-        // in PHP 1 == 1.0 is false
-        if (stringVersion.indexOf ('.') > 0) {
+        // in PHP 1 == 1.0 is true, but 1 === 1.0 is false
+        if (stringVersion.indexOf ('.') >= 0) {
             return parseFloat (stringVersion);
         }
         return parseInt (stringVersion);
     }
 
+    isRoundNumber (value) {
+        // this method is similar to isInteger, but this is more loyal and does not check for types.
+        // i.e. isRoundNumber(1.000) returns true, while isInteger(1.000) returns false
+        const isNumeric = (typeof value === 'number');
+        if (isNumeric) {
+            return (value % 1) == 0; // if we use `===` then in php it transpiles into fmod() which will not return true
```

Comment:
the thing is we have an eslint that enforces strict comparison 

```
 1796:32  error  Expected '===' and instead saw '=='  eqeqeq
```

Reply:
but what about my comment aside of that lines? `fmod() === 0` ends up to misbehave

Reply:
so, we have to either move this method into non-transpilable part, or don't transpile `%` to `fmod`. (and do as astt does).
please suggest

Reply:
@ttodua I don't think that's needed to move tot he non-transpilable part, let me take a look again

Reply:
ok thanks. so `fmod(x, x) === 0` returns false, that causes different result for this method in php